containers-template/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { Container, loadBalance, getContainer } from "@cloudflare/containers";
+
+export class MyContainer extends Container {
+  defaultPort = 8080;
```

Comment:
might consider adding comments to these options for users who are less familiar with these (and don't read the docs)